Lib/idlelib/iomenu.py

```diff
@@ -206,7 +212,21 @@ def save(self, event):
         if not self.filename:
             self.save_as(event)
         else:
+            # Check the time of most recent content modification so the
+            # user doesn't accidentally overwrite a newer version of the file.
+            if self.file_timestamp != os.stat(self.filename).st_mtime:
+                confirm = tkMessageBox.askokcancel(
```

Comment:
tkMessageBox does not exist.

Reply:
Fixed.  (It did when the original patch was written.)